upstream/jobs.go

```diff
@@ -352,3 +352,31 @@ func reconcileTable(ctx context.Context, config UpstreamConfig, table pushableTa
 		}
 	}
 }
+
+func ResetIsPushed(ctx context.Context) error {
+	overrides := map[string]string{
```

Comment:
can we make the interval configurble

Reply:
are we not missing config relationship ?

Reply:
> can we make the interval configurble

Done

> are we not missing config relationship ?

No. we don't need to override it since it has created,updated and deleted fields